### PR TITLE
feat: include sentinel image version in config

### PR DESCRIPTION
**Pull request by strophy**
This PR adds the Sentinel image version to config.

## Issue being fixed or feature implemented
Sentinel is the only service not yet included and versioned in config. It should be in config so it can be updated independently to mn-bootstrap updates by users directly using `mn config:set`

## What was done?
Added `core.sentinel.docker.image` setting and upgrade rules.

## How Has This Been Tested?
I was unable to test upgrade rules since I'm still not sure how semver parses `-dev` suffixes, see discussion [here](https://github.com/dashevo/mn-bootstrap/pull/252#discussion_r568102141).

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have added or updated relevant unit/integration/functional/e2e tests
- [x] I have made corresponding changes to the documentation

**For repository code-owners and collaborators only**
- [x] I have assigned this pull request to a milestone

### CODE REVIEW

File: src/config/configOptionMigrations.js [hunk 1]
```diff
@@ -23,6 +23,11 @@ module.exports = {
     // Set DashPay contract ID and block height for testnet
     lodashSet(options, 'platform.dashpay', systemConfigs.testnet.platform.dashpay);
 
+    return options;
+  },
+  '0.17.8': (name, options) => {
```
Comment (shumkov) [inline src/config/configOptionMigrations.js:28]: I guess it must be `0.18.0` since we merge it to `v0.18-dev` brach

File: docker-compose.yml [hunk 2]
```diff
@@ -15,7 +15,7 @@ services:
       - -masternodeblsprivkey=${CORE_MASTERNODE_OPERATOR_PRIVATE_KEY}
 
   sentinel:
-    image: dashpay/sentinel
+    image: ${CORE_DOCKER_IMAGE:?err}
```
Comment (shumkov) [inline docker-compose.yml:18]: `CORE_SENTINEL_DOCKER_IMAGE`

File: src/config/configOptionMigrations.js [hunk 3]
```diff
@@ -23,6 +23,11 @@ module.exports = {
     // Set DashPay contract ID and block height for testnet
     lodashSet(options, 'platform.dashpay', systemConfigs.testnet.platform.dashpay);
 
+    return options;
+  },
+  '0.18.0': (name, options) => {
+    lodashSet(options, 'core.sentinel.docker.image', 'dashpay/sentinel:1.5.0');
```
Comment (shumkov) [inline src/config/configOptionMigrations.js:29]: It won’t work ‘coz you don’t have sentinel object. So you need to set `systemConfigs.base.core.sentinel` with `core.sentinel` key. It’s better to test before :))

**Review by shumkov — CHANGES_REQUESTED** [review]

File: src/config/configOptionMigrations.js (hunk 3 above)
Comment (strophy) [reply src/config/configOptionMigrations.js:29]: I thought I can't test this unless it is released? I still don't understand how this function works, so I am literally guessing in these commits. Please show me how to test it.

Comment (shumkov) [reply src/config/configOptionMigrations.js:29]: Just set 0.18.0 in package json and run any command. Please don’t forget to bring version back before you commit.

**Comment by shumkov** [comment]
@strophy please resolve conflicts.

**Review by shumkov — APPROVED** [review]
Looks good 👍

**Pull request merged**